src/osm_configurator/view/toplevelframes/main_menu_frame.py

```diff
@@ -95,3 +134,32 @@ def __call_settings(self):
         Calls the settings-menu switching states
```

Comment:
can't make comment in corect line, idk why.
In line 130 don't use the .value, the method takes an enum as attribute

Reply:
same in Line 136